.gitignore

```diff
@@ -19,7 +19,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-
```

Comment:
And this change ought not to be in this PR

Reply:
I implemented this feedback